src/hb-ot-layout-private.hh

```diff
@@ -241,6 +241,7 @@ enum hb_unicode_props_flags_t {
   UPROPS_MASK_GEN_CAT	= 0x001Fu,
   UPROPS_MASK_IGNORABLE	= 0x0020u,
   UPROPS_MASK_FVS	= 0x0040u, /* MONGOLIAN FREE VARIATION SELECTOR 1..3 */
+  UPROPS_MASK_TAG       = 0x0080u, /* TAG characters */
```

Comment:
Yeah, I was thinking of changing the FVS to something like HIDDEN instead.  Let's keep the 0x0080 flag for future.